html-forms/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <div>Email</div>
+    <input type="email" name="email" />
+    <div>Why is coding cool?</div>
+    <select name="options">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select>
+    <div>Favorite Language (Choose One)
+      <br />
+      <input type="radio" name="language" value="html" />HTML
+      <input type="radio" name="language" value="CSS" />CSS
+      <input type="radio" name="language" value="Javascript" />Javascript
+    </div>
+    <div>would like to sign up for more information?
+      <br />
+      <input type="checkbox" name="information" value="Sign me up!" /> Sign me up!
+    </div>
+    <div>Comments</div>
```

Comment:
should be a label here with the for attribute pointing to an id attribute on the text area.